packages/calcite-components/src/components/input-date-picker/resources.ts

```diff
@@ -1,6 +1,15 @@
 export const CSS = {
   assistiveText: "assistive-text",
+  calendarWrapper: "calendar-wrapper",
+  calendarWrapperEnd: "calendar-picker-wrapper--end",
```

Comment:
☝️ should the value here omit the word `picker` to be more inline with the key pattern?

Reply:
Great catch! 🎣 Fixing.